script/testing/junit/InsertTest.java

```diff
@@ -156,4 +158,69 @@ public void test_2Tuple_NCS_2() throws SQLException {
 		 new int [] {11, 12, 0});
         assertNoMoreRows(rs);
     }
+    /*
+     * issue #706 fixed. Also need issue #724 for drop
+     * CREATE TABLE with a qualified namespace doesn't work as expected
+     */
+    // @Test
+    public void test_issue_706() throws SQLException {
+        String createSchemaSQL = "create schema foo;";
+        String createTableSQL = "create table foo.bar (id integer);";
+        String selectSQL = "SELECT * from pg_catalog.pg_class;";
```

Comment:
This code doesn't do execute anything.

Reply:
I'm testing the code coverage part and haven't finished testcases. This test case is commented out since I believe issue #724 is related with this issue, and SELECT and DROP are needed for testcases.